pkg/startup/startup.go

```diff
@@ -66,7 +67,8 @@ const (
 	// KubeadmConfigConfigMap is defined in k8s.io/kubernetes, which we can't import due to versioning issues.
 	KubeadmConfigConfigMap = "kubeadm-config"
 	// Rancher clusters store their state in this config map in the kube-system namespace.
-	RancherStateConfigMap = "full-cluster-state"
+	RancherStateConfigMap            = "full-cluster-state"
+	DEFAULT_AUTODETECT_POLL_INTERVAL = 60 * time.Minute
```

Comment:
I think this interval is probably too long. This means that potentially, when a node gets a new address over DHCP, that node would be out of operation for an hour. 

I was imagining a polling interval more on the order of 10 seconds or so so that changes are detected quickly with minimal interruption. There's not really a way to get _no_ interruption, because the address for the node is changing and routes will be need to be updated, but 10 seconds or so seems acceptable.

Reply:
Ideally we would be monitoring using something like this and only reconciling when we get a notification that an address has changed on the host: https://godoc.org/github.com/vishvananda/netlink#AddrSubscribe

Reply:
Thanks Casey. The latest change addresses this comment. PTAL.